### ISSUE TITLE
Feature request: mana_ssid_filter_file (Blacklist mode)

### ISSUE THREAD

**Issue by adde88** [sensepost/hostapd-mana#23]
If we enable **mana_ssid_filter_file** hostapd-mana  will only repsond to probe-requests for SSID's specified in our file, operating in a "whitelist-mode".

It would be awesome to also have the reverse option, like a "blacklist-mode", for this SSID-filter.
So that mana will respond to all probe-requests, except from the one(s) in the blacklist-file.

I'll be taking a deeper look into the source to see if this change is something i can manage to pull off, and then create a PR if wanted.
If not, consider this a feature request :)

 - Andreas Nilsen / @adde88

**Comment by singe** [comment]
Yeah I’ve been meaning to get to this. Just have some other critical path stuff for Defcon. Check src/ap/beacon.c for where the decision to prove or not is made. Gripping for “MANA” will help zoom in.

**Comment by adde88** [comment]
Thanks for the advice!  :)
I'll be looking into tonight then. Have plenty of time on my hand forward now, so i'd be happy to contribute with something like this.
